hack/lib/golang.sh

```diff
@@ -668,7 +666,7 @@ kube::golang::build_binaries() {
     kube::golang::build_kube_toolchain
 
     if [[ "${parallel}" == "true" ]]; then
-      kube::log::status "Building go targets for ${platforms[@]} in parallel (output will appear in a burst when complete):" "${targets[@]}"
+      kube::log::status "Building go targets for {${platforms[*]}} in parallel (output will appear in a burst when complete):" "${targets[@]}"
```

Comment:
this change wasn't really necessary, but now the platforms are all listed on one line, which is nice. e.g.

```
+++ [0816 17:05:44] Building go targets for {linux/amd64 linux/arm linux/arm64} in parallel (output will appear in a burst when complete):
    cmd/kube-dns
    cmd/kube-proxy
    cmd/kube-apiserver
    cmd/kube-controller-manager
    cmd/kubelet
    cmd/kubemark
    cmd/hyperkube
    federation/cmd/federation-apiserver
    federation/cmd/federation-controller-manager
    plugin/cmd/kube-scheduler
```
and
```
+++ [0816 17:02:26] Building go targets for {linux/amd64 linux/arm linux/arm64 linux/ppc64le} in parallel (output will appear in a burst when complete):
    cmd/kube-dns
    cmd/kube-proxy
    cmd/kube-apiserver
    cmd/kube-controller-manager
    cmd/kubelet
    cmd/kubemark
    cmd/hyperkube
    federation/cmd/federation-apiserver
    federation/cmd/federation-controller-manager
    plugin/cmd/kube-scheduler
```